playbooks/demo_copy_and_install_vios.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Copy the installation image for 3.1.0.10 version to HMC and install vios
```

Comment:
Install the VIOS version 3.1.0.10 after copying the image from remote server to Power HMC disk